tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -8,239 +8,142 @@ Feature: accept/decline shares coming from internal users
       | username |
       | user1    |
       | user2    |
+      | user3    |
```

Comment:
It's better to create a user in individual scenario if the user is not used in every scenario.

Reply:
okay..ThankYou